apps/dashboard/app/(app)/projects/[projectId]/deployments/components/controls/components/deployment-list-datetime/index.tsx

```diff
@@ -2,13 +2,27 @@ import { DatetimePopover } from "@/components/logs/datetime/datetime-popover";
 import { cn } from "@/lib/utils";
 import { Calendar } from "@unkey/icons";
 import { Button } from "@unkey/ui";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import { useFilters } from "../../../../hooks/use-filters";
 
+const TITLE_EMPTY_DEFAULT = "Select Time Range";
+
 export const DeploymentListDatetime = () => {
-  const [title, setTitle] = useState<string | null>("Last 12 hours");
+  const [title, setTitle] = useState<string | null>(TITLE_EMPTY_DEFAULT);
   const { filters, updateFilters } = useFilters();
 
+  // If none of the filters are set anymore we should reset the title
```

Comment:
@ogzhanolguncu we should do this for all time filters I think. Right now they still show the latest selected value (like "2 weeks"), even if it gets cleared